features/step_definitions/stepdefs.js

```diff
@@ -794,7 +796,7 @@ async function waitForOutboxActivity(
     const found = await findInOutbox(activity);
 
     if (found) {
-        return;
+        return found;
```

Comment:
This is required for `const foundInOutbox = await waitForOutboxActivity(foundActivity);` to work